lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_server/00_authorization_group/attestation_test_requirement_60.rb

```diff
@@ -0,0 +1,48 @@
+module CarinForBlueButtonTestKit
+  class AttestationTestCarinV2Requirement60_67 < Inferno::Test
+    title 'Supports SMART Core Capabilities'
+    description <<~DESCRIPTION
+      The Health IT Module supports the following “SMART Core Capabilities”:
+      - `launch-standalone`: support for SMART’s Standalone Launch mode
+      - `client-public`: support for SMART’s public client profile (no client authentication)
+      - `client-confidential-symmetric`: support for SMART’s confidential client profile
+      - `sso-openid-connect`: support for SMART’s OpenID Connect profile
+      - `context-standalone-patient`: support for patient-level launch context (requested by launch/patient scope, conveyed via patient token parameter)
+      - `permission-offline`: support for refresh tokens (requested by offline_access scope)
+      - `permission-patient`: support for patient-level scopes (e.g. patient Observation.read)
+      - `permission-user`: support for user-level scopes (e.g. user/Appointment.read)
+    DESCRIPTION
+    id :carin_server_requirement_60_67_attestation
+
+    verifies_requirements 'hl7.fhir.us.carin-bb_2.0.0@60',
```

Comment:
Looks like requirement 60 is in an automated test?

Reply:
included it here as well because this one attestation covers multiple requirements that are all within the same context in the IG, so including it should be minimal extra effort (if any) and provide consistent reference to the IG